flink-table/flink-sql-client/src/main/java/org/apache/flink/table/client/config/entries/ExecutionEntry.java

```diff
@@ -108,6 +108,10 @@
 
 	public static final String EXECUTION_CURRENT_DATABASE = "current-database";
 
+	private static final String EXECUTION_INTERACTIVE_VIEW_ENABLED = "interactive-view.enabled";
```

Comment:
I think I asked that in the past but why do we need an additional property? We have a property `result-mode` which could be extended by another enum value. Every additional config param should be well thought through because it increases the complexity for the user.

Reply:
result-mode describes how we organize the result (change-log or table). and this switch describes how we show the result to user(interactive or non-interactive). IMO they are two different dimensions.